cmake/external/googletest.cmake

```diff
@@ -43,8 +42,17 @@ if (NOT TARGET googletest_project)
                         EXCLUDE_FROM_ALL ON
                         PREFIX "${CMAKE_BINARY_DIR}/external/googletest"
                         INSTALL_DIR "${CMAKE_BINARY_DIR}/external"
-                        URL ${GOOGLE_CLOUD_CPP_GOOGLETEST_URL}
-                        URL_HASH SHA256=${GOOGLE_CLOUD_CPP_GOOGLETEST_SHA256}
+                        GIT_REPOSITORY
```

Comment:
Same here, you can use the URL as above to download a specific SHA as a tarball.